tests/unit/executors/indexers/test_binary_indexer.py

```diff
@@ -1,21 +1,10 @@
+import os
 from jina.flow import Flow
 from tests import random_docs
 
 
-# def test_binary_pb():
-#     num_docs = 100
-#     docs = list(random_docs(num_docs, jitter=50))
-#     with BinaryPbIndexer('test-shelf') as spi:
-#         spi.add(docs)
-#         spi.save()
-#
-#     with BinaryPbIndexer.load(spi.save_abspath) as spi:
-#         assert spi.size == num_docs
-#         for j in range(num_docs):
-#             assert spi.query(j) == docs[j]
-
-
-def test_binarypb_in_flow():
+def test_binarypb_in_flow(tmpdir):
```

Comment:
You could also use the default global fixture, which takes care about setting and deleting the environment variable via
```
@pytest.mark.parametrize('random_workspace_name', ['JINA_TEST_WORKSPACE_BINARY_PB'])
def test_binarypb_in_flow(test_metas):
    ....
```

Reply:
Done!